ooiui/templates/science/streams.html

```diff
@@ -21,6 +22,14 @@
   <script src="/js/compiled/index.js" type="text/javascript"></script>
 
   <script src="/js/compiled/svgplot.js" type="text/javascript"></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderMouseTouch.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderDraggable.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderBar.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderHandle.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderLabel.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSlider.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQDateRangeSliderHandle.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQDateRangeSlider.min.js'></script>
```

Comment:
Absolutely right, it's not for next week (this week) as most people are out.  But it will be done!